code/modules/projectiles/ammunition/bullets.dm

```diff
@@ -495,7 +495,7 @@
 
 /obj/item/ammo_casing/antim/lethal
 	name = "\"Red-Nose\" shell casing"
-	desc = "A SA 14.5mm Anti-Materiel \"Red-Nose\" shell."
+	desc = "A SA 14.5mm Anti-Materiel \"Red-Nose\" shell, punches organics harder than the standard round."
```

Comment:
```suggestion
	desc = "An SA 14.5mm Anti-Materiel \"Red-Nose\" shell, punches organics harder than the standard round."
```
Same quirk as above. "An 'ess-ayy'..." as pronounced. Also applies to all other SA 14.5mm ammo.